ocaml/default.nix

```
@@ -2150,6 +2150,27 @@ with oself;
   session = callPackage ./session { };
   session-redis-lwt = callPackage ./session/redis.nix { };
 
+  sherlodoc = buildDunePackage {
+    pname = "sherlodoc";
+    version = "0.2";
+    src = fetchFromGitHub {
+      owner = "art-w";
+      repo = "sherlodoc";
+      rev = "0.2";
+      sha256 = "sha256-MEYKtlVoSYZhh4ernon1FHGFykfeCmv6qQi+cyy3LX8=";
+    };
+    preBuild = ''
+      PATH="${lib.makeBinPath [menhir js_of_ocaml-compiler]}:$PATH"
+    '';
+    propagatedBuildInputs = [
+      result tyxml ppx_blob lwt fpath decompress cmdliner brr bigstringaf base64
+      odoc odoc-parser fmt
+    ];
+  };
+  sherlodoc-server = oself.sherlodoc.overrideAttrs (o: {
```

there's already `with oself;` at the top.

```suggestion
  sherlodoc-server = sherlodoc.overrideAttrs (o: {
```